reactnative/assets

```diff
@@ -0,0 +1 @@
+../assets
```

Comment:
isn't this a symlink?  should it be committed?

Reply:
Yes, it's a symlink and it should be committed in order to work on React Native.